Sources/ApolloCodegenLib/Templates/TemplateRenderer.swift

```diff
@@ -103,7 +103,7 @@ extension TemplateRenderer {
     \(ImportStatementTemplate.SchemaType.template(for: config))
 
     \(ifLet: detachedTemplate, { "\($0)\n" })
-    \(ifLet: namespace, template.wrappedInNamespace(_:), else: template)
+    \(ifLet: namespace, { template.wrappedInNamespace($0, accessModifier: accessControlModifier(target: target, definition: .namespace)) }, else: template)
```

Comment:
Not the prettiest but wrappedInNamespace requires to know the access modifier too now.